testsuite/tests/lf_skiplist/test_parallel.ml

```diff
@@ -1,4 +1,5 @@
 (* TEST
+   * no-tsan
```

Comment:
Caution: this test script does not run any tests. An empty script runs the default test but if you add a test clause, it does not run the default anymore.

Reply:
Fixed, thank you.